snippets/omt.snippets.json

```
@@ -0,0 +1,98 @@
+{
+    // Place your snippets for odt here. Each snippet is defined under a snippet name and has a prefix, body and
+    // description. The prefix is what is used to trigger the snippet and the body will be expanded and inserted. Possible variables are:
+    // $1, $2 for tab stops, $0 for the final cursor position, and ${1:label}, ${2:another} for placeholders. Placeholders with the
+    // same ids are connected.
+    // Example:
+    // "Print to console": {
+    //     "prefix": "log",
+    //     "description": "Log output to console",
+    //     "body": [
+    //         "console.log('$1');",
+    //         "$2"
+    //     ]
+    // }
+
+    // MODULE ENTRY POINT / DOCUMENT CONSTRUCTS
+    "Menu definition": {
+        "prefix": "menu",
+        "description": "Define sections of the application menu.\n\nUse in OMT modules only.",
+        "body": [
+            "menu:",
+            "    ${1:SectionName}:",
+            "    -   title: ${2:Menu item name}",
+            "        onSelect: |",
+            "            ${0:@LOG('snippet');}"
```

Je zou kunnen doen (voor de gein), dan pakt hij daar standaard de title mee, toch?

```suggestion
            "            ${0:@LOG('$2');}"
```